Contributing.md

```diff
@@ -15,13 +15,13 @@ Current maintainers of Stylus are:
 - Valeriy Chupurnov ([@xdan](https://github.com/xdan)).
 
 
-If you'll have any questions, feel free to mention us or use emails from our profiles to contact us.
+If you have any questions, feel free to contact us via our profile email or mentioning us in the repository.
 
 
 
 ## Code of Conduct
 
-Please note that this project is released with a [Contributor Code of Conduct](Code_of_Conduct.md). By participating in this project you agree to abide by its terms.
+Please note that this project is released with a [Contributor Code of Conduct](Code_of_Conduct.md). By participating in the project, you agree to abide by its terms.
```

Comment:
this is the same sentence

Reply:
Yeah, I didn't correct the meaning. I just simply think if we add comma it will make the sentence more readable :D